react/features/welcome/components/RootNavigationContainer.js

```diff
@@ -53,6 +55,13 @@ const RootNavigationContainer = ({ isWelcomePageAvailable }: Props) => (
                     component = { DialInSummary }
                     name = { screen.dialInSummary }
                     options = { dialInSummaryScreenOptions } />
+                <RootStack.Screen
```

Comment:
Let's move the entire navigation stuff to a standalone feature, since it doesn't belong in welcome or conference anymore.

Reply:
I will create a navigation feature and move everything there.

Reply:
❤️

Reply:
Moved everything to navigation feature.